core-lib/TestSuite/StreamTests.ns

```diff
@@ -0,0 +1,126 @@
+class StreamTests usingPlatform: platform testFramework: minitest = Value (
+| private TestContext = minitest TestContext.
+  private AsyncTestContext = minitest AsyncTestContext.
+  private actors      = platform actors.
+  private Exception   = platform kernel Exception.
+  private NotAValue   = platform kernel NotAValue.
+  private Vector      = platform kernel Vector.
+  private Array       = platform kernel Array.
+  private ObjectMirror = platform mirrors ObjectMirror.
+  private errorTestMessage = 'test exception 1'.
+  private Streams = platform streams.
+  private Files = platform files.
+  private fdes = Files FileDescriptor.
+  private filePath = Files FilePath.
+|
+)(
+  public class BasicStreamTests = TestContext (
+    | path testString|
+    path:: (filePath currentDirectory / 'tests' / 'streams') pattern.
+    testString:: 'This is a test String, it is written to a file via a 
+    readWriteStream, and then read and checked to make sure it works'.
+  )(
+
+    public testReadBinary = (
+      | fp cc t |
+      fp:: filePath for: path + '/Binary.blob'.
+      cc:: fp readStream.
+
+      assert: cc position = 0.
+      deny: cc atEnd.
+
+      t:: (cc next: (cc size)).
+
+      1 to: 20 do: [ :i|
+        assert: (t at: i) = i.
+      ].
+
+      assert: cc atEnd.
+      
+      cc close.
+    )
+
+    public testReadBinary2 = (
```

Comment:
What's the difference between the two tests?
What is their purpose?
I can't really tell from the method names.
And the assertions aren't self explanatory either.